### ISSUE TITLE
Tapping an IOUPreview split should launch read-only IOUConfirmationList

### ISSUE THREAD

**Issue by Julesssss** [Expensify/App#18640]
### Problem

We introduced the IOUPreview for bill splits [here](https://github.com/Expensify/Expensify/issues/270707). As the read-only IOUConfirmationList page wasn't ready we currently take the users to the IOUParticipants page when tapped. But this should be changed to launch the IOUConfirmationList as a read-only page where no edits can be made. 

### Solution

Yes please.

**Comment by Julesssss** [comment]
Still held

**Comment by Julesssss** [comment]
This is possibly unblocked now. Investigating today

**Comment by JmillsExpensify** [comment]
Yeah, I think we should take this one off-hold.

**Comment by Julesssss** [comment]
Okay, I just handed off another issue so will pick this one up.

**Comment by Julesssss** [comment]
Made a lot of progress today. [WIP PR](https://github.com/Expensify/App/pull/19390/files)

**Remaining tasks**
- Improve URL path
- Provide the action for amount details
- Use action to build participants, NOT the chatReport\
- Update the disabled fields styling (depending on the outcome of an ongoing discussion)

**Comment by Julesssss** [comment]
Updated the PR today to use the real action data. Then I merged Carlos' PR into mine as it had some similar changes that further improved the solution. [PR is in early review](https://github.com/Expensify/App/pull/19390/files) and I'll finish it tomorrow morning.

**Comment by Julesssss** [comment]
I just resubmitted the [PR](https://github.com/Expensify/App/pull/19390) for what is hopefully the final review

**Comment by sobitneupane** [comment]
@Julesssss Can you please assign me the issue. I reviewed the PR.

**Comment by Julesssss** [comment]
Assigned you @sobitneupane 👍

**Comment by Julesssss** [comment]
Hey @Christinadobrzyn, would you mind paying out for a C+ review from @sobitneupane? Thanks

**Comment by sobitneupane** [comment]
> To confirm, I assume this was an Internal job so there's no speed bonus?

Yup

Accepted the offer @Christinadobrzyn

**Comment by Christinadobrzyn** [comment]
Thanks! Paid you in Upwork. Are we good to close this?
